v2/bolt_store_test.go

```diff
@@ -414,3 +416,59 @@ func TestBoltStore_SetUint64_GetUint64(t *testing.T) {
 		t.Fatalf("bad: %v", val)
 	}
 }
+
+func TestBoltStore_MigratetoV2(t *testing.T) {
+
+	dir, err := ioutil.TempDir("", t.Name())
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+	defer os.RemoveAll(dir)
+
+	var sourceFile, destFile string
+	sourceFile = filepath.Join(dir, "/sourcepath")
+	destFile = filepath.Join(dir, "/destpath")
```

Comment:
You don't need to declare variables here:
```suggestion
	sourceFile := filepath.Join(dir, "sourcepath")
	destFile := filepath.Join(dir, "destpath")
```